### PR TITLE
fix(GRO-863): Article Q/A (1)

### PR DESCRIPTION
**Pull request by dzucconi**
Handles the bulk of article Q/A. There's still a bit more to do but I want to get these design/layout updates out.

Board: https://www.notion.so/artsy/6da224ab4ecd4833bd3280bc13a6a157?v=844809b11bc444f182d242b36b3641bc

### CODE REVIEW

File: src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+interface ArticleZoomGalleryResponsiveBoxProps {
+  aspectWidth: number
+  aspectHeight: number
+}
+
+/**
+ * - We can't create a grey placeholder box using `ResponsiveBox` because it has to be both width and height constrained.
+ * - We can't create a grey placeholder box using native CSS `aspect-ratio` because it is uncenterable using flexbox as of March 2022.
+ * - Instead: we just manually measure and create a grey placeholder box using JS.
+ */
```
Comment (dzucconi) [inline src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx:15]: Very annoying

Comment (dzucconi) [reply src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx:15]: Maybe I should use a native aspect ratio box and center it using JS 🤔

File: src/v2/Apps/Article/Components/ArticleBody.tsx [hunk 2]
```diff
@@ -78,6 +79,16 @@ const ArticleBody: FC<ArticleBodyProps> = ({ article }) => {
 
           <Spacer mt={4} />
 
+          {article.leadParagraph && (
+            <HTML
+              variant="sm"
+              html={article.leadParagraph}
+              maxWidth={OPTIMAL_READING_WIDTH}
```
Comment (damassi) [inline src/v2/Apps/Article/Components/ArticleBody.tsx:86]: 👌

File: src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  }, [aspectHeight, aspectWidth, height, ref, width])
+
+  return (
+    <Box ref={ref as any} width="100%" height="100%" display="flex">
+      <Box width={scaledWidth} height={scaledHeight} m="auto" bg="black10">
+        {children}
+      </Box>
+    </Box>
+  )
+}
```
Comment (damassi) [inline src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx:48]: This file is funny, but it does make sense

File: src/v2/Apps/Article/Components/Sections/ArticleSectionText.tsx [hunk 4]
```diff
@@ -4,6 +4,8 @@ import { ArticleSectionText_section } from "v2/__generated__/ArticleSectionText_
 import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components"
 
+export const OPTIMAL_READING_WIDTH = "65ch"
```
Comment (damassi) [inline src/v2/Apps/Article/Components/Sections/ArticleSectionText.tsx:7]: Did not know about `ch`

**Review by damassi — APPROVED** [review]
Looks great and learned a few things too 👍

**Pull request merged**